pylintrc

```diff
@@ -55,7 +55,7 @@ extension-pkg-whitelist=alsaaudio,pyA20
 # E1103: %s %r has no %r member (but some types could not be inferred) - fails to infer real members of types, e.g. in Celery
 # W0231: method from base class is not called - complains about not invoking empty __init__s in parents, which is annoying
 # R0921: abstract class not referenced, when in fact referenced from another egg
-disable=mixed-indentation,line-too-long,bad-continuation,missing-docstring,too-many-nested-blocks,too-many-branches,no-self-use,global-statement,too-many-statements
+disable=mixed-indentation,line-too-long,bad-continuation,missing-docstring,too-many-nested-blocks,too-many-branches,no-self-use,global-statement,too-many-statements,duplicate-code,relative-import
```

Comment:
Where do we have mixed indentation? That could (and should) be fixed easily I believe.